.github/workflows/updates/Hyper.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 webVer=$(get_release Jai-JAP/hyper-arm-builds)
-armhf_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_armv7l.deb"
+armhf_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${webVer}hyper_${webVer}_armv7l.deb"
 arm64_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_arm64.deb"
```

Comment:
this still has the wrong variable name. I know it was already wrong before but please fix it here as well

Reply:
I thought I fixed it.. Thanks for fixing it.